ios-base/Networking/Services/UserResource.swift

```diff
@@ -44,7 +59,31 @@ enum UserResource: TargetType {
           ]
         ]
       )
-    case .logout:
+    case .signup(let email, let password, let avatar64):
+      let picData = avatar64.jpegData(compressionQuality: 0.75) ?? Data()
+      let parameters = [
+        "user": [
+          "email": email,
+          "password": password,
+          "password_confirmation": password,
+          "image": picData.asBase64Param()
+        ]
+      ]
+      return requestParameters(parameters: parameters)
```

Comment:
maybe we can move cases implementation to helper functions so the switch reads better

Reply:
Yes, that's a good practice when things start getting complex